shop_table.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE orders (
+  id SERIAL PRIMARY KEY,
+  customer text,
+  date date
+);
+
+-- Then the table with the foreign key first.
+CREATE TABLE items (
+  id SERIAL PRIMARY KEY,
+  stock int,
+  name text,
+  price int,
+-- The foreign key name is always {other_table_singular}_id
+  order_id int,
```

Comment:
Does the item need to have an order id? I feel like the order needs to link to the item, so that the order knows which item it needs, so I think the foreign key should be in the order table.